app/templates/docs/scim.html

```diff
@@ -22,63 +22,61 @@ <h2 id="overview">Overview of SCIM</h2>
   <li><a target="_blank" href="https://tools.ietf.org/html/rfc7644">RFC7644</a> — Protocol, defining the behavior to expect from the API, and how to interact with it</li>
   <li><a target="_blank" href="https://tools.ietf.org/html/RFC7642">RFC7642</a> — Definitions, Overview, Concepts, and Requirements</li>
 </ul>
-<p>It allows an enterprise to provision and deprovision users and groups automatically across all the services they use (hosted and cloud-based). As soon as they create a user in their central identity system, a Trello user is created to match.
+<p>It allows a Trello user who is an admin of a <a target="_blank" href="https://trello.com/enterprise">Trello Enterprise</a> to provision and deprovision users and groups automatically across all the services they use (hosted and cloud-based). As soon as they create a user in their central identity system, a Trello user is created to match.
 SCIM also allows enterprises to sync between directories and Trello, to control the team membership of their users.</p>
 
 <h2 id="version">SCIM version</h2>
-<p>Trello supports SCIM version 2.0, at <a target="_blank" href="https://trello.com/scim/v2">https://trello.com/scim/v2</a>.</p>
+<p>Trello supports SCIM version 2.0, at <a target="_blank" href="https://api.trello.com/scim/v2">https://api.trello.com/scim/v2</a>.</p>
```

Comment:
Since `api.trello.com/scim/` is just redirecting to `trello.com/scim/` I'm just going to drop the `api.`.